include/mapnik/text/placements/combined.hpp

```diff
@@ -0,0 +1,70 @@
+/*****************************************************************************
+ *
+ * This file is part of Mapnik (c++ mapping toolkit)
+ *
+ * Copyright (C) 2015 Walid Ibrahim
+ *
```

Comment:
Copyright should be `Copyright (C) 2015 Artem Pavlenko` as per https://github.com/mapnik/mapnik/blob/master/docs/contributing.markdown